org.cryptomator.Cryptomator.yaml

```diff
@@ -0,0 +1,100 @@
+app-id: org.cryptomator.Cryptomator
+command: cryptomator
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --env=PATH=/app/bin/:/app/jre/bin/:/usr/bin/
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=session-bus
+  - --socket=x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.gtk.vfs
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: libfuse
+    config-opts:
+      - MOUNT_FUSE_PATH=/app/bin
+    post-install:
+      - install fusermount-wrapper.sh /app/bin/fusermount
+    sources:
+      - type: archive
+        url: https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
+        sha256: d0e69d5d608cc22ff4843791ad097f554dd32540ddc9bed7638cc6fea7c1b4b5
+      - type: file
+        path: build-aux/fusermount-wrapper.sh
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: cryptomator
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk17/bin
+        MAVEN_OPTS: -Dmaven.repo.local=.m2/repository
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+        VERSION: 1.6.5
+        REVISION_NO: '1'
+    build-commands:
+      - mvn clean package -DskipTests -Plinux
```

Comment:
At this stage you can do `-Dmaven.test.skip.exec`.

Reply:
Should be the same, see https://stackoverflow.com/a/21933970/4014509